apps/vr-tests-react-components/src/stories/Positioning.stories.tsx

```diff
@@ -570,3 +575,58 @@ storiesOf('Positioning', module)
     </Screener>
   ))
   .addStory('arrow', () => <Arrow />, { includeRtl: true });
+
+const ScrollJump = () => {
+  const popperRef = React.useRef<PopperRefHandle>({ updatePosition: () => null, setTarget: () => null });
+  const buttonRef = React.useRef<HTMLButtonElement>(null);
+  const [open, setOpen] = React.useState(false);
+
+  React.useEffect(() => {
+    if (open && buttonRef.current) {
+      buttonRef.current.focus();
+    }
+  }, [open]);
+
+  const { containerRef, targetRef } = usePopper({
+    popperRef,
+    position: 'above',
+    align: 'start',
+  });
+
+  return (
+    <>
+      <button ref={targetRef} onClick={() => setOpen(s => !s)}>
+        Target
+      </button>
+      {open && (
+        <Box ref={containerRef}>
+          Focusable element <button ref={buttonRef}>Focus me</button>{' '}
+        </Box>
+      )}
+    </>
+  );
+};
+
+storiesOf('Positioning - scroll', module).addStory('no jump', () => {
```

Comment:
separate category here because the `position: fixed;` centering for other stories doesn't play nicely with scrolling.

Adding `overflow: scroll` to the previous stories need an update of all the screenshots